Userland/Libraries/LibWeb/HTML/EventLoop/EventLoop.cpp

```diff
@@ -90,41 +91,51 @@ void EventLoop::process()
 {
     // An event loop must continually run through the following steps for as long as it exists:
 
-    // 1. Let taskQueue be one of the event loop's task queues, chosen in an implementation-defined manner, with the constraint that the chosen task queue must contain at least one runnable task. If there is no such task queue, then jump to the microtasks step below.
+    // 1. Let oldestTask be null.
+    OwnPtr<Task> oldest_task;
+
+    // 2. Let taskStartTime be the current high resolution time.
+    // NOTE: 'current high resolution time' in hr-time-3 takes a global object,
+    //       the HTML spec has not been updated to reflect this, let's use the shared timer.
+    // FIXME: This should maybe coarsen
```

Comment:
This comment isn't very clear

Reply:
I think I'll just remove the FIXME, since this value is not directly exposed to users anyway and will probably be cleared up whenever the noted issue with `current high resolution time` is resolved

Reply:
I removed the FIXME, upgraded the NOTE to a FIXME and referenced the spec issue I filed.